### PR TITLE
add wishlist product from plp and pdp

### PR DESCRIPTION
**Pull request by kibo-susantasamanta**

### CODE REVIEW

File: pages/category/index.vue [hunk 1]
```diff
@@ -141,14 +141,16 @@
             :show-add-to-cart-button="true"
             :score-rating="3"
             :max-rating="5"
-            wishlist-icon=""
-            is-in-wishlist-icon=""
-            :is-in-wishlist="false"
+            wishlist-icon="heart"
```
Comment (kibo-chandradeeptalaha) [inline pages/category/index.vue:144]: No need to send this

File: pages/category/index.vue [hunk 2]
```diff
@@ -141,14 +141,16 @@
...
+            is-in-wishlist-icon="heart_fill"
```
Comment (kibo-chandradeeptalaha) [inline pages/category/index.vue]: No need to send icons

File: pages/category/index.vue [hunk 3]
```diff
@@ -141,14 +141,16 @@
...
+            :is-in-wishlist="isInWishlist(product)"
             :regular-price="productGetters.getPrice(product).regular"
             :special-price="productGetters.getPrice(product).special"
```
Comment (kibo-chandradeeptalaha) [inline pages/category/index.vue]: Change this to 
:special-price="
              productGetters.getPrice(product).special &&
              $n(productGetters.getPrice(product).special, 'currency')
            "

**Review by kibo-chandradeeptalaha — CHANGES_REQUESTED** [review]
Please take a look

File: pages/category/index.vue (hunk 3 above)
Comment (kibo-susantasamanta) [reply pages/category/index.vue]: Done.

File: pages/category/index.vue (hunk 2 above)
Comment (kibo-susantasamanta) [reply pages/category/index.vue]: Done.

**Review by kibo-chandradeeptalaha — APPROVED** [review]

**Review by TriptiVermaXP — APPROVED** [review]

**Pull request merged**